### PR TITLE
iozone_windows:update iozone_cmd for ioeventfd test

### PR DESCRIPTION
**Pull request by aliang123**
generic.tests.iozone_windows:
1.set a default value for "disk_letter"
2.modify iozone_cmd for ioeventfd test

Signed-off-by:Aihua Liang <aliang@redhat.com>

ID:1412895

### CODE REVIEW

File: generic/tests/iozone_windows.py [hunk 1]
```diff
@@ -76,7 +76,10 @@ def get_driver():
                                        mountpoint=disk_letter)
     winutils = utils_misc.get_winutils_vol(session)
     cmd = params["iozone_cmd"]
-    iozone_cmd = re.sub("WIN_UTILS", winutils, cmd)
+    if re.search("WIN_UTILS", cmd):
+        iozone_cmd = re.sub("WIN_UTILS", winutils, cmd)
+    else:
+        iozone_cmd = cmd % winutils
```
Comment (suqinhuang) [inline generic/tests/iozone_windows.py]: you can use utils_misc.set_winutils_letter directly

**Comment by suqinhuang** [comment]
and you can update iozone_cmd in ioeventfd to 
iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f ${disk_letter}:\testfile"

File: generic/tests/iozone_windows.py [hunk 2]
```diff
@@ -76,7 +76,10 @@ def get_driver():
                                        mountpoint=disk_letter)
     winutils = utils_misc.get_winutils_vol(session)
     cmd = params["iozone_cmd"]
-    iozone_cmd = re.sub("WIN_UTILS", winutils, cmd)
```
Comment (xutian) [inline generic/tests/iozone_windows.py]: original code looks fine, why need to check it ?

**Comment by aliang123** [comment]
Test cmd:
  python ConfigTest.py --guestname=Win10 --platform=x86_64 --verbose=yes --testcase=ioeventfd.ioeventfd_on
Test Result:
  PASS 1-smp_2.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.ioeventfd.ioeventfd_on

**Comment by xutian** [comment]
looks fine, thanks @aliang123

**Pull request merged**